content/angular/step02.md

```diff
@@ -21,11 +21,15 @@ To start working on our todos list app, let's replace the code of the default st
 
 {{> DiffBox tutorialName="simple-todos-angular" step="2.3"}}
 
-Let's create a template for todosList component.
+Now we need to create a new directory called `imports`, a specially-named directory which will behave different than other directories in the project.  Files outside the `imports` directory will be loaded automatically when the Meteor server starts, while files inside the `imports` directory will only load when an `import` statement is used to load them.
```

Comment:
Small grammar issue - "which will behave different" should be "which will behave differently".